kube/kd/deployment.yaml

```diff
@@ -166,6 +166,8 @@ spec:
             - --resources=uri=/export/MTS*|roles=MPAM_EXPORT_USER
             - --resources=uri=/export/COMP*|roles=COMP_EXPORT_USER
             - --resources=uri=/export/FOI*|roles=FOI_EXPORT_USER
+            - --resources=uri=/export/TO*|roles=TO_EXPORT_USER
+            - --resources=uri=/export/BF*|roles=BF_EXPORT_USER
             - --resources=uri=/export/IEDET*|roles=IEDET_EXPORT_USER
             - --resources=uri=/export/somu/FOI*|roles=FOI_EXPORT_USER
             - --resources=uri=/export/somu/MPAM*|roles=MPAM_EXPORT_USER
```

Comment:
Do we need roles checks here for any contributions for TO and BF as well?

Reply:
I nearly did, but feel it is out of scope for the ticket. If it is required we should wait for the ticket, as far as I know they have not requested it.

Reply:
I have added the somu things user roles for TO and BF.